tests/unit/modules/test_asset.py

```diff
@@ -13,25 +14,242 @@
 )
 
 
+class TestDoDiffer:
```

Comment:
Would it be possible to add another test that has equal builds but one of the other fields is different?